### DISCUSSION TITLE
!perso

### DISCUSSION THREAD

**Discussion by DeadAngelV6** [Crownicles/Crownicles#664] [Suggestions fermées]
Ajouter une commande pour voir les perso de draftbot ? (Eviter les questions c'est qui gruffo etc etc) Comme ca aussi on a tout de centralisé ( vu que pas sur que le site soit à jour sur ca)

**Comment by Pietagorh** [comment]
Pas fan du nom de la commande
!lore peut-être ?

**Comment by Chevalier2** [comment]
Le nom lore me semble correct. D'autant que cela permettrait de réunir plusieurs choses, avec cette commande on pourrait peut-être avoir accès au lore sur le Kyutisme ainsi que les personnages ?

**Reply by Pietagorh** [reply]
Bonne idée
Mais faudrait être assez bref pour que les gens le lisent
Ou alors juste le lien vers la partie lore du wiki

**Comment by BastLast** [comment]
pas fan de la commande, ca réduirait grandement l'interet du mini event

**Reply by DeadAngelV6** [reply]
Quel mini event ?

**Comment by BastLast** [comment]
je trouve que c'est un peu le charme du truc que yai une limitation comme ca :)

**Comment by BastLast** [comment]
ptdr chui trop débile j'ai mal lu, du coup je refuse toujours l'idée mais ̂pour une autre raison, ya une page du guide qui donne cette info par contre question est-ce qu'on rajouterai pas un conseil avec un lien direct vers cette page ?
